node/service/src/chain_spec/moonriver.rs

```diff
@@ -28,8 +28,8 @@ use moonriver_runtime::{
 	currency::MOVR, AccountId, AuthorFilterConfig, AuthorMappingConfig, Balance, BalancesConfig,
 	CouncilCollectiveConfig, CrowdloanRewardsConfig, DemocracyConfig, EVMConfig,
 	EthereumChainIdConfig, EthereumConfig, GenesisConfig, InflationInfo, MaintenanceModeConfig,
-	ParachainInfoConfig, ParachainStakingConfig, Precompiles, Range, SchedulerConfig, SystemConfig,
-	TechCommitteeCollectiveConfig, WASM_BINARY,
+	MigrationsConfig, ParachainInfoConfig, ParachainStakingConfig, Precompiles, Range, Runtime,
```

Comment:
Importing `Runtime` here makes me think we should re-export the migrations from each runtime. And we should export a type alias that already has the generic runtime type filled in.